Lib/glyphsLib/builder/kerning.py

```diff
@@ -54,7 +54,12 @@ def _to_ufo_kerning(self, ufo, kerning_data):
             ufo.kerning[left, right] = kerning_val
 
     seen = {}
-    for classname, glyph, is_left_class in reversed(class_glyph_pairs):
+    # Sort by the 3rd tuple member: `is_left_class`, so that pairs with
+    # a class on the right are seen first, and thus have priority when
+    # solving ambiguous cases, as per the UFO spec at:
+    # http://unifiedfontobject.org/versions/ufo3/kerning.plist/#exception-conflict-resolution
+    for classname, glyph, is_left_class in sorted(
+            class_glyph_pairs, key=lambda pair: pair[2]):
```

Comment:
maybe the lambda function here could return a tuple with the first item being `pair[2]`, and the second and third items being `pair[0], pair[1]`. So that within the group that have the same `is_left_class` pairs are sorted alphabetically.

Reply:
alternatively we could change `class_glyph_pairs` list's tuples to already be in the correct order, so instead of

```
for classname, glyph, is_left_class in sorted(..., key=lambda ...)
```

we could have

```
for is_left_class, classname, glyph in sorted(...)
```